src/data/machines.js

```diff
@@ -46,22 +46,126 @@ export const gpuTypes = [
   { name: 'NVIDIA Tesla V100', type: 'nvidia-tesla-v100', numGpus: 1, maxNumCpus: 12, maxMem: 78, price: 2.4800, preemptiblePrice: 0.7400 },
   { name: 'NVIDIA Tesla V100', type: 'nvidia-tesla-v100', numGpus: 2, maxNumCpus: 24, maxMem: 156, price: 4.9600, preemptiblePrice: 1.4800 },
   { name: 'NVIDIA Tesla V100', type: 'nvidia-tesla-v100', numGpus: 4, maxNumCpus: 48, maxMem: 312, price: 9.9200, preemptiblePrice: 2.9600 },
-  { name: 'NVIDIA Tesla V100', type: 'nvidia-tesla-v100', numGpus: 8, maxNumCpus: 96, maxMem: 624, price: 19.8400, preemptiblePrice: 5.9200 }
+  { name: 'NVIDIA Tesla V100', type: 'nvidia-tesla-v100', numGpus: 8, maxNumCpus: 96, maxMem: 624, price: 19.8400, preemptiblePrice: 5.9200 },
+  { name: 'NVIDIA Tesla P100', type: 'nvidia-tesla-p100', numGpus: 1, maxNumCpus: 16, maxMem: 104, price: 1.4600, preemptiblePrice: 0.4300 },
+  { name: 'NVIDIA Tesla P100', type: 'nvidia-tesla-p100', numGpus: 2, maxNumCpus: 32, maxMem: 208, price: 2.9200, preemptiblePrice: 0.8600 },
+  { name: 'NVIDIA Tesla P100', type: 'nvidia-tesla-p100', numGpus: 4, maxNumCpus: 96, maxMem: 624, price: 5.8400, preemptiblePrice: 1.7200 }
+]
+
+export const zonesToGpus = [
+  { name: 'ASIA-EAST1-A', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-p100', 'nvidia-tesla-k80'] },
+  { name: 'ASIA-EAST1-B', validTypes: ['nvidia-tesla-k80'] },
+  { name: 'ASIA-EAST1-C', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-v100', 'nvidia-tesla-p100'] },
+  { name: 'ASIA-EAST2-A', validTypes: [] },
+  { name: 'ASIA-EAST2-B', validTypes: [] },
+  { name: 'ASIA-EAST2-C', validTypes: [] },
+  { name: 'ASIA-NORTHEAST1-A', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'ASIA-NORTHEAST1-B', validTypes: [] },
+  { name: 'ASIA-NORTHEAST1-C', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'ASIA-NORTHEAST2-A', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'ASIA-NORTHEAST2-B', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'ASIA-NORTHEAST2-C', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'ASIA-NORTHEAST3-A', validTypes: [] },
+  { name: 'ASIA-NORTHEAST3-B', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'ASIA-NORTHEAST3-C', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'ASIA-SOUTH1-A', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'ASIA-SOUTH1-B', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'ASIA-SOUTH1-C', validTypes: [] },
+  { name: 'ASIA-SOUTHEAST1-A', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'ASIA-SOUTHEAST1-B', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-p4'] },
+  { name: 'ASIA-SOUTHEAST1-C', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-p4'] },
+  { name: 'ASIA-SOUTHEAST2-A', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'ASIA-SOUTHEAST2-B', validTypes: ['nvidia-tesla-t4'] },
+  { name: 'ASIA-SOUTHEAST2-C', validTypes: [] },
+  { name: 'AUSTRALIA-SOUTHEAST1-A', validTypes: ['nvidia-tesla-t4', 'nvidia-tesla-p4'] },
+  { name: 'AUSTRALIA-SOUTHEAST1-B', validTypes: ['nvidia-tesla-p4'] },
+  { name: 'AUSTRALIA-SOUTHEAST1-C', validTypes: ['nvidia-tesla-p4'] },
```

Comment:
I see a different GPU type on https://cloud.google.com/compute/docs/gpus/gpu-regions-zones (screenshot below)

```suggestion
  { name: 'AUSTRALIA-SOUTHEAST1-C', validTypes: ['nvidia-tesla-p100'] },
```

![Screen Shot 2021-10-22 at 5 08 59 PM](https://user-images.githubusercontent.com/5438223/138524633-2e56d0b5-fd0d-434c-a176-436a27f79c75.jpg)

Reply:
Fixed